components/clarity-vscode/README.md

```diff
@@ -1,50 +1,69 @@
-# Clarity for VSCode
+# Clarity for Visual Studio Code
 
-Clarity is a decidable smart contract language that optimizes for predictability and security. Smart contracts allow developers to encode essential business logic on a blockchain.
+Clarity is a **decidable** smart contract language that optimizes for predictability and security. Developed by Hiro, Clarity enables smart contract developers to encode essential business logic on a blockchain.
 
-This extension aims to help developer write safe and clean Clarity code, it includes syntax highlighting, auto-completion, linting, debugging and [safety checks](https://www.hiro.so/blog/new-safety-checks-in-clarinet),.
+A programming language is referred to as **decidable** if you can know with certainty, from the code itself, what the program will do. Clarity is was designed to be intentionally Turing incomplete, thereby avoiding the problem of `Turing complexity` (the number of steps it takes for a turing machine to run an algorithm on a given input size). This allows for complete static analysis of the entire call graph of a given smart contract, while Hiro's support for types and type checkers can eliminate whole classes of bugs like unintended casts, reentrancy bugs, and reads of uninitialized values.
```

Comment:
I don't think it's necessary to go into too much details about what Clarity is, at least not in the intro of the readme.

Instead, I think some popular extensions tend to list al the implemented features, allowing developers to quickly understand what to expect from the extension

Reply:
+1 to what Hugo says.

@trichards-hiro: This content, in particular, if it can speak to our extension and its value prop, that'd help the readers tremendously. Clarity, as a language, can be ignored entirely from here.

Reply:
removed the unnecessary Clarity section from intro